src/coreclr/vm/threads.cpp

```diff
@@ -865,11 +865,8 @@ void DestroyThread(Thread *th)
         th->UnmarkThreadForAbort();
     }
 
-    if (g_fEEShutDown == 0)
```

Comment:
Unnecessary check (also, the way it is written triggered a build error after changing g_fEEShutDown  to Volatile)